redisson/src/main/java/org/redisson/RedissonLocalCachedMap.java

```diff
@@ -166,6 +167,11 @@ public boolean equals(Object obj) {
             return true;
         }
 
+        @Override
+        public int hashCode() {
+            return Objects.hash(getValue());
```

Comment:
@molekyla I'm sorry, but JDK 1.7 API usage is not allowed to save compatibility with 1.6